Makefile

```diff
@@ -3,15 +3,7 @@
 SHELL=/bin/bash -o pipefail
 
 build:
-	@pack builder create heroku/buildpacks:20 --config buildpacks-20/builder.toml --pull-policy always
 	@pack builder create heroku/builder:20 --config builder-20/builder.toml --pull-policy always
 	@pack builder create heroku/builder:22 --config builder-22/builder.toml --pull-policy always
-	@pack builder create heroku/builder-classic:22 --config builder-classic-22/builder.toml --pull-policy always
+	@pack builder create heroku/builder:24 --config builder-24/builder.toml --pull-policy always
 	@pack builder create salesforce-functions --config salesforce-functions/builder.toml --pull-policy always
-
-# The salesforce-functions image isn't published to Docker Hub, so is intentionally absent here.
-publish: build
```

Comment:
I've removed this publish Makefile target since we don't (and shouldn't) be publishing locally - particularly now there are many more details to take into account given multi-arch (which means even if the GHA pipeline went down, this current Makefile target wouldn't be sufficient to roll out a release anyway).